### PR TITLE
Add spring petclinic demo

### PR DESCRIPTION
**Pull request by vpa1977**
#### Changes proposed in this pull request:
 - This MR adds a Spring Petclinic demo which can be explored by the user
   Note: due to building within docker container, the build time is rather long, 
   so not including demo into CI.

- [*] I have signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

*Picture of a cool ROCK:*

### CODE REVIEW

File: demos/petclinic/readme.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Spring PetClinic
+
+This docker file contains [Spring PetClinic](https://github.com/spring-projects/spring-petclinic) sample built on top of chiselled-jre container.
```
Comment (cjdcordeiro) [inline demos/petclinic/readme.md]:
```suggestion
This Dockerfile runs the [Spring PetClinic](https://github.com/spring-projects/spring-petclinic) sample on the `ubuntu/chiselled-jre` container image.
```

File: demos/petclinic/readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+# Running the sample
+
+Execute:
+
+`` docker build -t petclinic . && docker run -p 8080:8080 --tmpfs /tmp:exec petclinic ``
+
+Navigate to http://localhost:8080 to explore the demo.
```
Comment (cjdcordeiro) [inline demos/petclinic/readme.md]:
```suggestion
Give it a few minutes to build the sample application and then navigate to http://localhost:8080 to explore the demo.
```

**Review by cjdcordeiro — APPROVED** [review]
super nice! thanks. It lgtm, just renamed the README file and suggested 2 text nitpicks

File: demos/petclinic/readme.md (hunk 1 above)
Comment (vpa1977) [reply demos/petclinic/readme.md]: I've applied those manually, github probably did not like renamed file =)

**Pull request merged**